.travis.yml

```diff
@@ -63,11 +34,34 @@ env:
       I/BFT0MbnR6JVCZiPV7TCWPgY1gvgZ6TEEIKGqauDMUBdL8ZK6I='
     - secure: e0yxVfwVW61d3Mi/QBOsY6Rfd1mZd3VXUd9xNRoz/fkvQJRuVwDe7oG3NOuJ4LZzvMw7BJ+zpDV9D8nKhAyPEEOgpkkMHUB7Ds83pHG4qSMzm4EAwBCadDLXCQirldz8dzN5FAqgGucXoj5fj/p2SKOkO6qWIZveGr8pdBJEG1E=
 
-jobs:
-  fail_fast: true
+matrix:
+  fast_finish: true
 
-  include:
+cache:
+  yarn: true
 
+stages:
+  - basic test
+  - additional tests
+  - name: deploy
+    if: type IN (push)
+
+before_install:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
```

Comment:
Are these two lines still needed?

```js
  - export DISPLAY=:99.0
  - sh -e /etc/init.d/xvfb start
```

I don't believe we still require anything running in a virtual frame buffer. I suspect old browser testing needed this.

Reply:
I suspect they are not needed any more

Reply:
I was assuming that maybe Firefox still needed that, so I left it in there for now.

Reply:
looks like this is indeed still needed (see https://travis-ci.org/emberjs/ember.js/jobs/463512118)

I've dropped the commit that removes it again. I suggest merging this and then we can iterate forward.

Reply:
Oh, I didn't realize we had a smoke test for FF usage using xvfb. Future work could likely just use https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Headless_mode and avoid that. 

But that shouldn't hold up this PR, thanks for investigating.